package.json

```diff
@@ -115,13 +115,13 @@
         "liquidjs": "^10.7.0",
         "mitt": "~3.0.1",
         "mongodb": "~4.17.1",
-        "mqtt": "~4.3.7",
+        "mqtt": "~5.3.5",
         "mssql": "~8.1.4",
         "mysql2": "~3.6.2",
         "nanoid": "~3.3.4",
         "node-cloudflared-tunnel": "~1.0.9",
         "node-radius-client": "~1.0.0",
-        "nodemailer": "~6.6.5",
+        "nodemailer": "~6.9.9",
```

Comment:
[Changelog](https://github.com/nodemailer/nodemailer/blob/master/CHANGELOG.md) looks fine for this one